.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,73 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+# GitHub recommends pinning actions to a commit SHA.
+# To get a newer version, you will need to update the SHA.
+# You can also reference a tag or branch, but the action may change without warning.
+
+name: Build Docker images
+
+on:
+  push:
```

Comment:
No value here? Is this equivalent to `**`?

Reply:
Yep! -  https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#using-a-single-event